src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -226,10 +229,16 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} error - Error object
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
+    errorCallback = (error: Error): void => {
```

Comment:
Would this be `ElementsXhrError` now, as well? It looks like it's been updated in the child sidebars.

Reply:
👍 good catch